fe/fe-core/src/main/java/com/starrocks/server/Session.java

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.server;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+
+// used for temporary tables to store all temporary table ids created by a session,
+// maintain the mapping relationship of databaseId -> tableName -> tableId.
+public class Session {
+    private static final Logger LOG = LogManager.getLogger(Session.class);
+    private UUID id;
+
+    private Map<Long, Map<String, Long>> temporaryTables;
```

Comment:
give some comment for this map. Also you can use Table directly.

Reply:
> give some comment for this map. Also you can use Table directly.

I have considered it, but in the current metadata, it seems that Table is only managed by Database. Do we need to respect this design? If not needed, I think we can use Table here directly. @imay